nomt/src/options.rs

```diff
@@ -136,6 +139,13 @@ impl Options {
     pub fn preallocate_ht(&mut self, preallocate_ht: bool) {
         self.preallocate_ht = preallocate_ht;
     }
+
+    /// Sets the size of the leaf cache in MiB.
```

Comment:
This is rounded down to the nearest page size, no?

Reply:
yes! I will update the comment now!